cli/chaosli/README.md

```diff
@@ -13,6 +13,9 @@ The Chaos Controller CLI, `chaosli`, is meant to help create a more user friendl
 
 Download the `chaosli` binary for your operating system and architecture [from the latest release](https://github.com/DataDog/chaos-controller/releases/latest).
 
+##### Requirements (important):
+- Have `go` installed (v. 1.11 or greater) and the local `$GOPATH` environment variable setup
```

Comment:
Can we just require 1.16 or higher here? Why support a lower version for this CLI than we want to support for the controller?

Reply:
I see the CLI as an executable that shouldn't require the chaos-controller code/tech nor its requirements to run - so I set it as the lowest possible value 
But yes we could 100% ask for 1.16 that's definitely future-proof

Reply:
I agree that's a very sensible point, but it's more that I don't want to install or think about any golang versions that I didn't already have to? 1.11 is EOL

Reply:
Done